application-templates/typescript/service/src/connector/actions.ts

```diff
@@ -0,0 +1,130 @@
+import { ByProjectKeyRequestBuilder } from '@commercetools/platform-sdk/dist/declarations/src/generated/client/by-project-key-request-builder';
+
+const CART_UPDATE_EXTENSION_KEY = 'myconnector-cartUpdateExtension';
+const CART_DISCOUNT_TYPE_KEY = 'myconnector-cartDiscountType';
+
+export async function createCartUpdateExtension(
+  apiRoot: ByProjectKeyRequestBuilder,
+  applicationUrl: string
+): Promise<void> {
+  const list = await apiRoot
+    .extensions()
+    .get({
+      queryArgs: {
+        where: `key = "${CART_UPDATE_EXTENSION_KEY}"`,
+      },
+    })
+    .execute();
+
+  if (list.body.results.length > 0) {
+    const result = list.body.results[0];
```

Comment:
Same here. Instead of `result`, why not use `extension`?

Reply:
Fixed. Thanks 💪